SecretManager/src/V1/DisableSecretVersionRequest.php

```diff
@@ -30,6 +30,19 @@ class DisableSecretVersionRequest extends \Google\Protobuf\Internal\Message
      * Generated from protobuf field <code>string etag = 2 [(.google.api.field_behavior) = OPTIONAL];</code>
      */
     private $etag = '';
+    /**
+     * @param string $name Required. The resource name of the [SecretVersion][google.cloud.secretmanager.v1.SecretVersion] to disable in the format
+     *                     `projects/&#42;/secrets/&#42;/versions/*`. Please see
+     *                     {@see SecretManagerServiceClient::secretVersionName()} for help formatting this field.
+     *
+     * @return \Google\Cloud\SecretManager\V1\DisableSecretVersionRequest
```

Comment:
We should probably label the `build` methods with `@experimental` as well, so they're considered "beta", even while being in a GA package.

Reply:
Good idea.

Reply:
resolving as I've pushed a version with the local fix.